### PR TITLE
LPD-20247 Updated styles of Solutions Search Page

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -5,16 +5,50 @@
...
+		font-size: 0.8125rem;
+		overflow: hidden;
+		text-overflow:ellipsis;
+		white-space: nowrap;
+		white-space: nowrap;
+		width: fit-content;
+	}
+	
+	.card-image-title-container{
+		padding:016px;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:21]:
```suggestion
	.card-image-title-container {
		padding: 16px;
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 2]
```diff
@@ -5,16 +5,50 @@
...
+	}
 
-	.adt-solutions-search-results .solutions-search-results-card:hover {
-		color: var(--black);
+	.developer-name-text{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:24]:
```suggestion
	.developer-name-text {
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 3]
```diff
@@ -5,16 +5,50 @@
...
-	.adt-solutions-search-results .card-image-title-container .image-container {
-		height: 3rem;
-		min-width: 3rem;
+	.image-container {
+		min-height: 200px;
+		min-width:293px;
+		overflow:hidden;
+	}
+	
+	.solution-search-image{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:38]:
```suggestion
	.solution-search-image {
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 4]
```diff
@@ -5,16 +5,50 @@
...
+		min-height: 200px;
+		min-width:293px;
+		object-fit:contain;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:41]:
```suggestion
		min-width: 293px;
		object-fit: contain;
```

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 5]
```diff
@@ -5,16 +5,50 @@
...
 	}
 
+	.solution-search-results-card{
+		border-radius:10px;
+		border: 1px solid #E7EFFF;
+		min-height:456px;
+		min-width:293px;
+		overflow:hidden;
+	}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:50]:
```suggestion
	.solution-search-results-card {
		border-radius: 10px;
		border: 1px solid #E7EFFF;
		min-height: 456px;
		min-width: 293px;
		overflow: hidden;
	}
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 6]
```diff
@@ -36,30 +70,31 @@
...
+	.solution-search-results-card .card-image-title-container .developer-name {
+		color: #545d69;
+	}
+
 	.adt-solutions-search-results .labels .category-label {
 		background-color: #ebeef2;
 		color: #545D69;
 		font-size: smaller;
+		overflow: hidden;
+		text-overflow:ellipsis;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:82]:
```suggestion
		text-overflow: ellipsis;
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**